build/cqs.tsv

```diff
@@ -198,6 +204,7 @@ org.apache.camel:camel-netty4	2.15.1	test
 org.apache.cassandra:cassandra-all	3.0.0	test
 org.apache.curator:curator-test	2.7.1	test
 org.apache.kafka:kafka_2.11	test:0.8.2.1	test
+org.apache.kafka:kafka_2.11	test:0.9.0.1	test
```

Comment:
@jnh5y we've merged some PRs without checking the cqs - we're going to need to do a diff and check what needs to be added.
going forward, it might be good to add a check to the travis build that runs the build-cqs and checks that this file is correct so that we see CQs when they get added to a PR.